lib/releases.js

```diff
@@ -79,12 +80,20 @@ const contributorsSentence = ({ commits, pullRequests }) => {
 const getCategoriesConfig = ({ config }) => {
   const categoriesConfig = config.categories.reduce((acc, category) => {
     acc[category.label] = category
+    if (category.labels && category.labels.length) {
```

Comment:
Would it make sense to arrayify `category.label` so that `category.label` and `category.labels` can be treated the same way? 
E.g.: `const labels = !!category.label ? [category.label] : category.labels`

Reply:
I'd prefer to concatenate the two arrays 😆